pkg/executor/build.go

```diff
@@ -627,7 +627,7 @@ func fetchExtraStages(stages []config.KanikoStage, opts *config.KanikoOptions) e
 			}
 			// This must be an image name, fetch it.
 			logrus.Debugf("Found extra base image stage %s", c.From)
-			sourceImage, err := util.RetrieveRemoteImage(c.From, opts)
+			sourceImage, err := util.RetrieveRemoteImage(c.From, &opts.SecureOptions)
```

Comment:
`opts.SecureOptions` is being passed as a pointer in many places where I don't see any modifications happening to the object. Is there an advantage to passing it as a pointer when it is such a small struct?

Reply:
emm，It seem be unnessasery， I will change it

Reply:
opts.SecureOptions is being passed as struct now